### PR TITLE
[MOD] Fix of Board header and Context menu tests

### PR DESCRIPTION
**Pull request by annaslodkowska**

### CODE REVIEW

**Review by MarekPawlowskiQA — APPROVED** [review]
lgtm :)

File: src/test/java/com/intive/template/pages/ContextMenuPage.java [hunk 1]
```diff
@@ -13,11 +13,11 @@
...
 public class ContextMenuPage extends BasePage {
 
-    @FindBy(how = How.XPATH, using = "//*[@id=\"__next\"]/div/div[1]/div[2]/div/div/button/span")
+    @FindBy(how = How.XPATH, using = "//a[@data-testid='board-tile'][1]//div[@data-testid='context-menu']/button")
     WebElement triggerButton;
-    @FindBy(how = How.XPATH, using = "//div[2]//div[1]//div[1]//nav[1]")
+    @FindBy(how = How.XPATH, using = "//a[@data-testid='board-tile'][1]//div[@data-testid='context-menu']/nav")
     WebElement openContextMenu;
-    @FindBy(how = How.XPATH, using = "/html/body/div/div/div[1]/div[2]/div/div/nav/ul")
+    @FindBy(how = How.XPATH, using = "//a[@data-testid='board-tile'][1]//div[@data-testid='context-menu']/nav/ul")
```
Comment (MarekPawlowskiQA) [inline src/test/java/com/intive/template/pages/ContextMenuPage.java:20]: Hmm yo could try to put it to real list and than in function, try to for each and return proper element. Not mandatory just, idea to check.

**Review by KrzysztofGajQA — APPROVED** [review]
lgtm :)

File: src/test/java/com/intive/template/pages/ContextMenuPage.java (hunk 1 above)
Comment (annaslodkowska) [reply src/test/java/com/intive/template/pages/ContextMenuPage.java:20]: I'll try to do it in next PR.

**Pull request merged**